dids/diddht/diddht.go

```diff
@@ -67,12 +31,12 @@ func (rec *dhtDIDRecord) DIDDocument(didURI string) *didcore.Document {
 
 	// Now create the did document
 	for name, data := range rec.records {
-
 		switch {
 		case strings.HasPrefix(name, "_k"):
-			vMethod, err := UnmarshalVerificationMethod(data)
-			if err != nil {
+			var vMethod didcore.VerificationMethod
+			if err := UnmarshalVerificationMethod(data, &vMethod); err != nil {
 				// TODO handle error
+				continue
```

Comment:
Return it?

BTW Go now has multiple error support via `errors.Join()`, so you could also accumulate all the errors during validation.

Reply:
I'm not sure what the right behavior is here yet. It might be that we just skip this particular vm vs failing completely. @mistermoe thoughts?